docs/pages/spec.mdx

```diff
@@ -418,6 +418,41 @@ Properties accessible via the TestBench value:
 
 ## Built-in Functions
 
+### moved(old_path, new_path)
```

Comment:
wonderful

the expectation is that the .zen file should already be updated to new_path, right? Should we add some warnings/errors if either `old_path` still exists in the schematic or `new_path` does not exist?

Reply:
Yeah, warnings are a good idea. Added both.